source/base/DateTime.ooc

```diff
@@ -14,22 +14,251 @@
 * You should have received a copy of the GNU Lesser General Public License
 * along with this software. If not, see <http://www.gnu.org/licenses/>.
 */
-TimeSpan: cover {
-	_ticks: Int64
-	ticks ::= this _ticks
-	init: func@ (=_ticks)
-	kean_base_timeSpan_getTicks: unmangled func -> Int64 { this _ticks }
-	new: unmangled(kean_base_timeSpan_new) static func ~API (ticks: UInt64) -> This { This new(ticks) }
+
+use ooc-base
+import os/Time
+
+DateTimeData: class {
```

Comment:
Also, glad it can be a cover. In case no one told you, covers are like structs, i.e. stack allocated, so you don't have to free them yourself, which is nice. ooc isn't as good at cleaning things up when they go out of scope as, say, C++ is.